src/c_api/c_predict_api.cc

```diff
@@ -444,6 +538,20 @@ int MXPredGetOutputShape(PredictorHandle handle,
   API_END();
 }
 
+int MXPredGetOutputType(PredictorHandle handle,
+                        mx_uint out_index,
+                        int* out_dtype) {
+  MXAPIPredictor* p = static_cast<MXAPIPredictor*>(handle);
+  API_BEGIN();
+  CHECK_LT(out_index, p->out_arrays.size())
+    << "Index exceed number of outputs";
```

Comment:
nit: Can we make the message more easy to comprehend?

Reply:
okay sure.